src/Auth0.OidcClient.Shared/Auth0Client.cs

```diff
@@ -95,13 +95,10 @@ private string CreateTelemetry()
 #elif WINDOWS_UWP
             var platform = "uwp";
 #endif
-            var version = GetType().GetTypeInfo().Assembly.GetCustomAttribute<AssemblyFileVersionAttribute>()
-                .Version;
+            var sdkVersion = typeof(Auth0Client).GetTypeInfo().Assembly.GetName().Version;
 
-            var telemetryString = $"{{\"name\":\"oidc-net\",\"version\":\"{version}\",\"platform\":\"{platform}\"}}";
-            var telemetryBytes = Encoding.UTF8.GetBytes(telemetryString);
-
-            return Convert.ToBase64String(telemetryBytes);
+            var agentJson = $"{{\"name\":\"oidc-net\",\"version\":\"{sdkVersion.Major}.{sdkVersion.Minor}.{sdkVersion.Revision}\",\"env\":{{\"platform\":\"{platform}\"}}}}";

```

Comment:
Could this end up with a blank `platform`? If so, I'd only add if we have one.

Reply:
No, the platform is defined inline.  If somebody added a new platform and didn't write the necessary line of code they'd get a compiler error.